api-reference/10 UI Widgets/dxScheduler/1 Configuration/scrolling.md

```diff
@@ -5,7 +5,8 @@ inheritsType: dxSchedulerScrolling
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures scrolling.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Scrolling allows a user to browse data left outside the current viewport. The widget provides several scrolling modes detailed in the [mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.
```

Comment:
The widget contains several scrolling modes detailed in the [mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.